pkl-core/src/test/files/LanguageSnippetTests/input/classes/class3.pkl

```diff
@@ -8,5 +8,5 @@ class Person {
 person = new Person {
   name = "Pigeon"
   // typo
-  adress = "Howdy St."
+  address_ = "Howdy St."
```

Comment:
My general preference is to add a `_` to a token to make it not match -- it's equivalent to a typo for anything that's parsing for a perfect match, but it doesn't upset dictionary based checks which reduces the number of false positives that a spell-checking tool needs to tolerate. (I didn't change `zipp` in a similar instance, but I'd be more than happy to do so.)

Reply:
This is fine; the spelling error is orthogonal to what this test is about anyways.